docs/intermediate-inputs.md

```diff
@@ -27,40 +27,46 @@ The workflow will pause at a `Suspend` node, and user will be able to update par
 [![Approval Example Demo](https://img.youtube.com/vi/eyeZ2oddwWE/0.jpg)](https://youtu.be/eyeZ2oddwWE)
 
 ```yaml
-
-entrypoint: cicd-pipeline
-templates:
-   - name: cicd-pipeline
-     steps:
-        - - name: deploy-pre-prod
-            template: deploy
-        - - name: approval
-            template: approval
-        - - name: deploy-prod
-            template: deploy
-            when: '{{steps.approval.outputs.parameters.approve}} == YES'
-   - name: approval
-     suspend: {}
-     inputs:
-        parameters:
-           - name: approve
-             default: 'NO'
-             enum:
-                - 'YES'
-                - 'NO'
-     outputs:
-        parameters:
-           - name: approve
-             valueFrom:
-                supplied: {}
-   - name: deploy
-     container:
-        image: 'argoproj/argosay:v2'
-        command:
-           - /argosay
-        args:
-           - echo
-           - deploying
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Should maybe link to the example file and vice-versa? So that they're kept in-sync

Reply:
Refers to each other!